frontend/src/pages/home.jsx

```diff
@@ -0,0 +1,162 @@
+                                ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+                                laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit 
+                                in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat c
+                                upidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                            </p>
+                            <br/>
+                        </div> 
+                        <div className="cell large-offset-4" style={{position:"absolute", display: "flex", marginTop:"150px" }}>
+                            <img src={Triangle} alt=""/>
+                        </div>
+                    </div>   
+                </div>     
+            </div>   
+            <div style={{background:"#FFEBF3"}}>
+                <div className="feature">
+                    <div className="cell small-7">
+                        <h2>
+                            Featured Stories
+                        </h2>
+                        <h6 style={{color:"#6E6D7A"}}>
+                            Read the impact stories from around the world
+                        </h6>
+                        <br/>
+                        <ul className="list__item-container large-offset-1">
+                            <li className="list__item">
+                                <BlogCard/>
+                            </li>
+                            <li className="list__item">
+                                <BlogCard/>
+                            </li>
+                            <li className="list__item">
+                                <BlogCard/>
+                            </li>
+                        </ul>
+                        <div style={{marginTop:"82px"}}>
+                        <a className="button__big button__big--hollow">
+                            SEE ALL STORIES
+                        </a>
+                        </div>
+                    </div>
+                </div>
+                <div style={{background:"linear-gradient(180deg, #FFEBF3 0%, #FFFDF6 100%)"}}>
+                    <div className="feature" style={{height:"542px"}}>
+                        <div className="cell small-7">
+                            <h2>
+                                Our Partners
+                            </h2>
+                            <h6 style={{color:"#6E6D7A"}}>
+                                We thank all our wonderful partners for all the support
+                            </h6>
+                            <br/>
+                            <ul className="list__item-container large-offset-1 centreitems">
+                                <li className="list__item">
+                                    <img src={GoogleAd} width="254.41px" height="80px"  alt="Google Ads"/>
```

Comment:
Never give fixed widths and heights. Everything should be relative (some exceptions are there, like logo). If you give fixed dimensions, it will break on smaller screens.

Reply:
I understand that the width is taken care of the parent element, eg. by using `cell large-4`, but what about height?

Reply:
Okay, got it 👍